homeassistant/components/light/hue.py

```diff
@@ -130,9 +134,18 @@ def update_lights():
 
         for light_id, info in api_states.items():
             if light_id not in lights:
-                lights[light_id] = HueLight(int(light_id), info,
-                                            bridge, update_lights,
-                                            bridge_type=bridge_type)
+
+                if _CONFIGURING['ignore_unreachable']:
+                    lights[light_id] = HueLight(int(light_id), info,
```

Comment:
This is a very weird `if` statement. You can just pass the value of ignore_unreachable as the value for the unreachable parameter.